### PR TITLE
chatterino: Add version 2.1.5

### PR DESCRIPTION
**Pull request by vhns**
This PR adds the Chatterino2 App Manifest to the scoop-extras bucket.

### CODE REVIEW

**Comment by vhns** [comment]
Fixed issue with trailling whitespaces caused by tabs being used instead of spaces.

**Comment by vhns** [comment]
Fixed dependency on vcredist. Instead of suggesting its install through "notes:" I added it to the dependency list.

**Comment by vhns** [comment]
Closed PR accidentally. Sorry if I'm filling this up with too many things.

**Comment by linsui** [comment]
I do not know is vcredist2017 and 2013 is needed. I can open this app without them but they are required in https://github.com/Chatterino/chatterino2 .
Add autoupdate. Add architecture. Some other fixes.

<details>
<summary>chatterino.json</summary>

```
{
    "version": "2.1.5-4",
    "homepage": "https://chatterino.com",
    "description": "Chat client for twitch chat",
    "license": "MIT",
    "architecture": {
        "64bit": {
            "url": "https://chatterino.com/download/2.1.5-4/Chatterino%20Portable.zip",
            "hash": "b3326f45f7d31f5f0b767b4805732f5e8192d0a141d61ccd86b367d41edb32f0"
        }
    },
    "extract_dir": "chatterino2",
    "bin": "chatterino.exe",
    "shortcuts": [
        [
            "chatterino.exe",
            "Chatterino"
        ]
    ],
    "persist": [
        "Settings",
        "Cache"
    ],
    "suggest": {
        "Microsoft Visual C++ Redistributable 2017": "extras/vcredist2017",
        "Microsoft Visual C++ Redistributable 2013": "extras/vcredist2013"
    },
    "checkver": {
        "url": "https://chatterino.com/download/",
        "regex": ">([\\d.-]+)/<",
        "reverse": true
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://chatterino.com/download/$version/Chatterino%20Portable.zip"
            }
        },
        "extract_dir": "chatterino$majorVersion"
    }
}

```

</details>

File: bucket/chatterino.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.1.5-4",
+    "homepage": "https://chatterino.com",
+    "description": "Chat client for twitch chat",
```
Comment (Ash258) [inline bucket/chatterino.json]:
```suggestion
    "description": "Twitch chat client",
```

File: bucket/chatterino.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        [
+            "chatterino.exe",
+            "Chatterino"
+        ]
+    ],
+    "persist": [
+        "Settings",
+        "Cache"
+    ],
+    "suggest": {
```
Comment (Ash258) [inline bucket/chatterino.json]: Move suggest before `architecture` property

File: bucket/chatterino.json (hunk 1 above)
Comment (vhns) [reply bucket/chatterino.json]: Added change to a single commit including other requested change. (Moving suggest before architecture)

File: bucket/chatterino.json (hunk 2 above)
Comment (vhns) [reply bucket/chatterino.json]: Included change in a single commit with another change. (Change of description)

Comment (vhns) [reply bucket/chatterino.json]: Added to a commit.

**Comment by vhns** [comment]
Added suggested changes.

**Pull request merged**